v1.2/drop-sequence.md

```diff
@@ -4,13 +4,13 @@ summary:
 toc: false
 ---
 
-<span class="version-tag">New in v1.2:</span> The `DROP SEQUENCE` [statement](sql-statements.html)...
+<span class="version-tag">New in v2.0:</span> The `DROP SEQUENCE` [statement](sql-statements.html) removes a sequence from a database.
 
 <div id="toc"></div>
 
 ## Required Privileges
 
-The user must have the...
+The user must have the `CREATE` [privilege](privileges.html) on the parent database.
 
 ## Synopsis
 
```

Comment:
Parameter names are inconsistent (e.g., seqname vs. sequenceName vs. name). Do we care / should we standardize across the sequence docs?

Reply:
They seem to be standardized in [functions.md](https://github.com/cockroachdb/cockroach/blob/0832bc5ed19e0654d131400e7a7fe9c82f1d5221/docs/generated/sql/functions.md#sequence-functions) as `sequence_name`; not sure where they're inconsistent

Reply:
updated.